client/extensions/woocommerce/lib/countries/US.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export default () => {
+	return (
+		{
+			code: 'US',
+			name: translate( 'United States of America' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **45** times:
`translate( 'United States' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).